docs/guides/validation/validation_actions/how_to_trigger_slack_notifications_as_a_validation_action.md

```diff
@@ -66,7 +66,7 @@ Open the `.yml` configuration file in `great_expectations/checkpoints` that corr
 Add the `send_slack_notification_on_validation_result` Action to the `action_list` section of the configuration. Make sure the following section exists in the Checkpoint configuration.
 
 <Tabs
-  groupId="webhook-or-app"
+  groupId="webhook-or-app2"
```

Comment:
Is this change intentional?

Reply:
It is.  Changing the group ID causes the two tab blocks to no longer be linked when a user changes tabs on one of them.  This prevents changing tabs in the lower tab group to cause the text to "jump" due to the earlier tab group also changing tabs and resizing.